### ISSUE TITLE
New error messages should indicate custom block name

### ISSUE THREAD

**Issue by ToonTalk** [jmoenig/Snap#2935]
While the following is great progress

![untitled script pic (38)](https://user-images.githubusercontent.com/2138696/145800278-d3f1540f-9190-41ed-8890-10a4f62ddc92.png)

shouldn't it also mention the name of the custom block that caused this error?

**Comment by cycomachead** [comment]
Yes. But I don’t know how to get that info. We don’t really have a normal stack trace I could use. 

And building one got hard so I never finished it. (It’s not like we don’t have the caller info, it’s just not always that easy to grab.)

-- 
Michael Ball
From my iPhone
michaelball.co
